File: bench/ceremony/03-csv-filter/task.go
package main

import (
    "encoding/csv"
    "fmt"
    "strings"
)

func main() {
    reader := csv.NewReader(strings.NewReader("name,active\nAda,true\nLin,false\nGrace,true"))
    rows, err := reader.ReadAll()
    if err != nil {
        panic(err)
    }
    for _, row := range rows[1:] {
        if row[1] == "true" {
            fmt.Println(row[0])
        }
    }
}

File: bench/ceremony/03-csv-filter/task.py
import csv
import io


def main() -> None:
    text = "name,active\nAda,true\nLin,false\nGrace,true"
    for row in csv.DictReader(io.StringIO(text)):
        if row["active"] == "true":
            print(row["name"])


main()
